tools/jsnap2py

```diff
@@ -24,7 +24,8 @@ def msg_change(x):
     if data:
         data = data[0]
         if data[0] in ['id', 'ID']:
-            msg = "{{%s_%s}}" % (data)
+            data = (data[0], int(data[1])-1)
+	    msg = "{{%s_%s}}" % (data)
```

Comment:
@vnitinv this is how the code looks in the tools/jsnap2py file. Not sure why does it show an indent problem here.
```
if data:
        data = data[0]
        if data[0] in ['id', 'ID']:
            data = (data[0], int(data[1])-1)
	    msg = "{{%s_%s}}" % (data)
        elif data[0] in ['PRE', 'POST']:
            msg = '{{%s["%s"]}}' % (data)
        else:
            msg = '{{POST["%s"]}}' % data.strip()
        return msg.lower()
```